src/components/TreeBreadcrumb.vue

```diff
@@ -74,13 +74,23 @@ export default {
       default: false
     }
   },
+  data() {
+    return {
+      pathSeparator: this.$core.config.get('pathSeparator')
```

Comment:
What happen if the pathSeparator is not configured (or wrongly defined)? do we have a fallback?

Reply:
`$config.get` accepts a default value: `this.$core.config.get('pathSeparator', '/')`

Reply:
Good remark